tests/unit/Common/Internal/Authentication/SharedAccessSignatureHelperTest.php

```diff
@@ -233,4 +233,377 @@ public function testGenerateAccountSharedAccessSignatureToken()
             $this->assertEquals($testCase[8], urlencode($actualSignature));
         }
     }
+
+    public function testGenerateServiceSharedAccessSignatureToken()
+    {
+        // Setup
+        $accountName = "phptests";
+        $accountKey = "WaZvixrkMok53QDmj8Tc99+BV6vO9cCOJNsdm+wD9QVEScwl8c1eYPcQ182ndNFqxX1+SEKs18SmOxh8OpzIUg==";
```

Comment:
It is, I was using the same info that the Account SAS PR was using. It got changed later. I'll have to update all these references. Thanks